README.md

```diff
@@ -13,7 +8,33 @@ SomaScan data deliverable. Data in this file is **not** intended for
 biological analysis purposes or to provide any metrics for SomaScan data in
 general.
 
-### ADAT File Format
+---------------------
+
+## Files 
+
+* example_data.adat
+* example_data_v4.1_plasma.adat 
+
+
+## Installation
+
+The example ADAT files in this repository can be retrieved in one of two ways:
+
+1) Cloning the repo to your local machine
```

Comment:
In markdown, enumerated lists can be designated like this:
```
1. foo
1. bar
1. baz
```
and they'll come out numbered in sequence once rendered ... this way you don't have to re-number if the list changes.